data/en/videos.json

```diff
@@ -0,0 +1,466 @@
+{
```

Comment:
Is this file generated? If so, how?

Reply:
Also, what other metadata is available for videos?

Reply:
This file was generated using [online API explorer](https://developers.google.com/youtube/v3/docs/search/list?apix_params=%7B%22part%22%3A%22snippet%2Cid%22%2C%22channelId%22%3A%22UCfkRcekMTa5GA2DdNKba7Jg%22%7D) and additionally remapped according to our needs. We will find the way to write script that will generate this.

Reply:
We can get tags and description. Will add it to generation script.

Reply:
We can get description and tags from API. Will provide these fields in generation scripts